planner/core/collect_column_stats_usage.go

```diff
@@ -270,17 +272,33 @@ func (c *columnStatsUsageCollector) collectFromPlan(lp LogicalPlan) {
 	}
 }
 
+func set2slice(set map[model.TableColumnID]struct{}) []model.TableColumnID {
```

Comment:
Since it's not a member function of a struct, we'd better use a more specific name.
Or let us make it a member function of the `columnStatsUsageCollector`.

Reply:
Hide `set2slice` to function `CollectColumnStatsUsage`.